i18n/it/taboos.po

```diff
@@ -69,337 +71,421 @@ msgstr ""
 
 msgid ""
 "Mutated. Permanent. Limit 1 [[Covenant]] per deck.\n"
 "[reaction] When you resolve a [bless] token during a skill test, exhaust Ancient Covenant: Do not reveal another token as part of this [bless] token's effect."
 msgstr ""
+"Mutato. Permanente. Limite di 1 carta <b><i>Sodalizio</i></b> per mazzo.\n"
+"[reaction] Quando risolvi un segnalino [bless] durante una prova di abilità, esaurisci il Sodalizio Antico: Non rivelare un altro segnalino come parte dell'effetto di questo segnalino [bless]."
 
 msgid ""
 "Mutated. While David Renfield has at least 1 doom on him, you get +1 [willpower].\n"
 "[fast] Exhaust David Renfield: You may place 1 doom on David Renfield. Gain 1 resource for each doom on David Renfield (to a maximum of 3 resources)."
 msgstr ""
+"Mutato. Finché ci sono 1 o più indizi su David Renfield, ricevi +1 [willpower].\n"
+"[fast] Esaurisci David Renfield: Puoi collocare 1 destino su David Renfield. Ottieni 1 risorsa per ogni destino su David Renfield (massimo 3 risorse)."
 
 msgid ""
 "As an additional cost to play A Watchful Peace, search the chaos bag and/or cards in play for a total of 5 [bless] tokens and return them to the token pool.\n"
 "Mutated. Fast. Play when the \"draw encounter cards\" step of the mythos phase would begin. Skip this step of the mythos phase. Remove A Watchful Peace from the game."
 msgstr ""
+"Come costo aggiuntivo per giocare la Pace Guardinga, cerca nel sacchetto del caos e/o tra le carte in gioco un totale di 5 segnalini [bless] e rimettili nella riserva dei segnalini.\n"
+"Mutato. Veloce. Gioca solo quando stai per iniziare il passo \"ogni giocatore pesca 1 carta incontro\" della fase dei Miti. Salta questo passo della fase dei miti. Rimuovi dal gioco la Pace Guardinga."
 
 msgid ""
 "As an additional cost to play Hallow, return a total of 10 [bless] tokens to the token pool from the chaos bag or sealed on cards in play.\n"
 "Mutated.\n"
 "Remove 1 doom from any card in play. Remove Hallow from the game."
 msgstr ""
+"Come costo aggiuntivo per giocare Consacrare, rimetti nella riserva dei segnalini un totale di 10 segnalini [bless] dal sacchetto del caos e/o sigillati sulle carte in gioco.\n"
+"Mutato.\n"
+"Rimuovi 1 destino da qualsiasi carta in gioco. Rimuovi dal gioco Consacrare."
 
 msgid ""
 "Mutated. Permanent. Limit 1 per deck. Purchase at deck creation.\n"
 "<b>Forced</b> - When your first turn of the game begins: One at a time, play up to 5 [[Item]] assets from your hand, reducing the cost of each by 1. During this turn, you have 3 fewer actions to take."
 msgstr ""
+"Mutato. Permanente. Limite di 1 per mazzo. Acquistabile solo durante la composizione del mazzo.\n"
+"<b>Obbligo</b> Quando il tuo primo turno della partita inizia: Uno alla volta, gioca fino a 5 supporti <b><i>Oggetto</b></i> dalla tua mano, riducendo di 1 il costo in risorse di ciascuno di essi. Possiedi 3 azioni in meno da effettuare durante questo turno."
```

Comment:
@Zoltirico looks like the closing <b><i> tags are flipped here (the linter will now run on your PRs, so it will flag things like this automatically going forward)